styles.css

```diff
@@ -1,21 +1,36 @@
 :root {
   --projectPurple: #6070ff;
   --headerColor: #172b4d;
+  --worksBold: #344563;
+  --worksLight: #7a869a;
+  --gridBackground: #e5e5e5;
+  --liButtonBackground: #ebebff;
+  --projectButtonColor: #396df2;
+  --languageBoxColor: #f7f7f9;
+  --pressedButtonColor: #2230d2;
 }
 
+/* GLOBALS */
+
 header {
   width: 100%;
-  height: 100px;
+  height: 50px;
   display: flex;
   justify-content: space-between;
-  margin-top: 40px;
+  margin-top: 1px;
+  align-items: center;
+  position: fixed;
```

Comment:
- [ ] For a better user experience we could avoid using `position: fixed` in the navbar. 👍🏽 
- [ ] Kindly we could fix the position of the logo and hamburger button in the navbar. 👽 

- Your design 

![image](https://user-images.githubusercontent.com/58642949/185724228-e31413a5-7b66-4e30-9a82-90f387300ac5.png)

- Figma template

![image](https://user-images.githubusercontent.com/58642949/185724311-c0fa0c41-45da-4404-a6d8-0454e02c9ce8.png)